app/main/views/direct_award.py

```diff
@@ -15,6 +16,22 @@
     get_valid_page_or_1, validate_and_return_updater_request, drop_all_other_fields)
 
 
+def get_project_by_id(project_id):
+    """During the transitional period, project_id may be either the internal or the public ID of the project."""
+    if project_id.isdigit():
+        project = DirectAwardProject.query.filter(DirectAwardProject.id == project_id).first()
+        if project:
+            return project
+
+    try:
+        project_public_id = PublicIdMixin.decode(project_id)
+
+    except ValueError:
+        abort(404)
+
+    return DirectAwardProject.query.filter(DirectAwardProject.public_id == project_public_id).first_or_404()
+
+
```

Comment:
Could probably abstract this into a general "get transitional id" function, which could be `partial`-ized for actual use and revived any time we needed to do another transition, but that's a future concern

Reply:
Yeah, I'd probably leave that for next time.